web/src/js/__tests__/components/ContentView/ContentLoaderSpec.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import ContentLoaderGenerator from '../../../components/ContentView/ContentLoader'
```

Comment:
Speaking in React terms, it's a [Higher-Order Component](https://facebook.github.io/react/docs/higher-order-components.html), like react-redux's `connect`. So maybe just call it `contentLoader` or `withContentLoader`? I have no strong preference, but we should do the same as we do in the actual non-test code. Please feel free to adjust both :)